bool wakeup
string asr_result
float32 angle
